LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -201,7 +201,7 @@ private async Task<bool> ExecuteC2DOperationAsync(Message cloudToDeviceMessage,
                 this.logger.LogDebug("done processing '{OperationName}' on cloud to device message, id: '{MessageId}'.", operationName, messageId);
                 return true;
             }
-            catch (OperationCanceledException ex) when (ExceptionFilterUtility.True(() => this.logger.LogError(ex, "'{OperationName}' failed on cloud to device message (id: {MessageId}) with error: '{ExceptionMessage}'.", operationName, messageId, ex.Message)))
+            catch (OperationCanceledException) when (ExceptionFilterUtility.True(() => this.logger.LogError("'{OperationName}' timed out on cloud to device message (id: {MessageId}).", operationName, messageId)))
```

Comment:
pls always pass the original exception to the logger. The stack traces where the timeout happened can be crucial.

Reply:
Done in 6fcc890f3a924576e042a9b5fc4c635d4bed2343 (as well as other occurrences in the same file)